inspirehep/modules/workflows/workflows/article.py

```diff
@@ -218,19 +212,34 @@
 
 CHECK_IF_SUBMISSION_AND_ASK_FOR_APPROVAL = [
     IF_ELSE(
-        is_record_relevant,
-        [halt_record(
-            action="hep_approval",
-            message="Submission halted for curator approval.",
-        )],
+        is_record_merged,
         [
-            reject_record("Article automatically rejected"),
-            stop_processing
+            IF(
+                is_there_any_conflict,
+                [
+                    submit_to_the_curator,
+                    halt_record(
+                        action="hep_approval",
+                        message="Submission halted for curator approval.",
+                    )
+                ]  # for the manual resolution of the conflicts
```

Comment:
> First thing: What do we do when there are conflicts? Should we put the article in the halted state and wait for the approval? Should we create a new_action for this specific case?

You should halt the workflow, you can reuse the same action that we have for the article one, as the halt is for the same reason, it needs curator approval/review.

> If there are no conflicts, should we simulate the fact that an article has been accepted from the moment that we are referring to a merged article? Should we handle other things in addition to this, like the fact that an article is core or not?

If there are no conflicts, then the update should go though, the merge itself should have taken care of picking up the correct field from the root or the update, so no further action should be done on it, unless I'm missing something...